packages/app/src/LogTable.tsx

```diff
@@ -377,7 +390,7 @@ export const RawLogTable = memo(
               {info.getValue<string>()}
             </span>
           ),
-          size: 150,
+          size: columnSizeStorage[column] ?? 150,
```

Comment:
I saw that the linter wasn't happy about this missing from the dep array which I'd like to have fixed, but noticed that when added the resizing no longer worked.

The side-effect of this as it is today is if you hit reset column widths, it doesn't actually reset column widths until the page is refreshed which is a pretty odd experience.

I'm wondering if you have any ideas on how to workaround those problems?

Reply:
Good call out, I have added `columnSizeStorage` to the deps array and the resize seemed to work fine for me. Is there set of steps to recreate it?

Reply:
I must've been doing something weird then - this PR works as is which is awesome!